README.md

```diff
@@ -13,44 +13,56 @@ The project has the following dependencies:
+* The `ConsumerAccessControlFilter.class` the `ProviderSecurityConfig.class`, the `SubscriberSecurityConfig.class` and the `PublisherSecurityConfig.class` extends the `AccessControlFilter.class` which is imported by the client-library dependency and responsible for setting the security level based on the `application.properties`.
+  -  `ConsumerAccessControlFilter.class` is effective only when your consumer is a web-server and `server.ssl.enabled` property is set to true. This filter is responsible for validating whether the received HTTPS request is coming from one of the local cloud's clients based on its certificate. *(**Look for the 'TODO' mark** within this class if you want to implement additional access rules.)*
+  -  `ProviderAccessControlFilter.class` is doing the same as described in the consumer case, but is effective only when `server.ssl.enabled` property is set to true and `token.security.filter.enabled` property is set to false. When `token.security.filter.enabled` property is set to true, then `ProviderTokenSecurityFilter.class` is effective which is validating whether a token is received within the HTTPS request and whether it is a valid one ore not. *(**Note** that the token is created by the Authorization Core System during the orchestration process and the consumer have to put it into its HTTPS request as a query parameter.)*
+  -  `PublisherAccessControlFilter.class` is doing the same as described in the provider case, but when `token.security.filter.enabled` property is set to true, then `PublisherTokenSecurityFilter.class` is effective and is doing the same as described in the provider case.
+  -  `SubscriberAccessControlFilter.class` is doing the same as described in the provider case, but when `token.security.filter.enabled` property is set to true, then `SubscriberTokenSecurityFilter.class` is effective.
+* The `SubscriberTokenSecurityFilter.class` checking the requested target Uri and if it is not registered in the applicaction.properties as notification Uri then validating whether a token is received within the HTTPS request and whether it is a valid one ore not.
+* The `SubscriberNotificationAccessControlFilter.class` checking the requested target Uri and if it is not registered in the applicaction.properties as notification Uri then validating whether the requester is and allowed Core System or not. If the requester is not allowed, throws an AuthExeption with `" is unauthorized to access "` text in the message body. *(**Note:** By default only the `EVENT HANDLER CORE SYSTEM` is allowed.)* 
```

Comment:
typos: 
missing IS before checking
URI
'not' is not necessary
application.properties
an instead of and
missing it before throws

Reply:
fixed